lng/english.lng.php

```diff
@@ -299,7 +299,6 @@
 $lng['admin']['deactivated'] = 'Deactivated';
 $lng['admin']['deactivated_user'] = 'Deactivate user';
 $lng['admin']['sendpassword'] = 'Send password';
-$lng['admin']['ownvhostsettings'] = 'Own vHost-settings';
```

Comment:
Why did you remove that? it's still in use

Reply:
Was deleted accident